### DISCUSSION TITLE
New PR: Robynpy - Robyn powered by Python.

### DISCUSSION THREAD

**Discussion by michaelwexler** [facebookexperimental/Robyn#1184] [General]
Just pointing out for the sidebar watchers that @sumane81 has opened a PR ([https://github.com/facebookexperimental/Robyn/pull/1183](https://github.com/facebookexperimental/Robyn/pull/1183)) with some pretty monumental changes, opening up the Robyn "engine" into an API layered approach which should really free up how many things you can do with your data either in Python or R, whichever your environment prefers.   I can see in the PR that others have added work here as well, so thanks to all the rest as well (just to name a few, @alxlyj, @shivkanthb, @dhavalpatel624624, @sumalreddy17, @Marco-Premier, and probably more I missed.)

Thanks for all the work, and excited to see this change!

**Comment by sumane81** [comment]
Thanks for noticing. I along with others mentioned are from Meta. We rewrote Robyn in Python with better, modular architecture. Watch out for press release on Robyn Python version from Meta soon. :)
